### DISCUSSION TITLE
NoFasel Extensions...(Read the comment)

### DISCUSSION THREAD

**Discussion by Heus-Sueh** [N0-0NE-Dev/NoFasel#25] [Ideas]
I think it would be better to adopt the same model as aniyomi, where do you have these extensions that are the sites that you scraped and the user downloads these extensions to access the content of a certain site
![Screenshot_20230219-165225_Aniyomi_01](https://user-images.githubusercontent.com/98482469/219972095-e4cc3e64-d829-4919-b079-1785a3bd914f.png)

**Comment by MOHDTAIYEB** [comment]
How do we add extensions, for me the app is stuck on loading loop

**Reply by abadawe-dev** [reply]
> How do we add extensions, for me the app is stuck on loading loop

The app does not support extensions, it does not work in some countries, that is why it does not load, see #20.

**Reply by abadawe-dev** [reply]
I am working on scraping a different website to get the app to work on other countries,

**Reply by MOHDTAIYEB** [reply]
Thank you 
And sorry for my ignorance
I'm pretty new to github and foss community

**Comment by iproazad** [comment]


**Reply by abadawe-dev** [reply]
What?
